controllers/contentlibrary/utils/controller_builder.go

```diff
@@ -126,7 +128,7 @@ func (r *Reconciler) Reconcile(
 	ctx = pkgcfg.JoinContext(ctx, r.Context)
 	ctx = ovfcache.JoinContext(ctx, r.Context)
 
-	logger := ctrl.Log.WithName(r.Kind).WithValues("name", req.String())
+	logger := logr.FromContextOrDiscard(ctx)
```

Comment:
Why not `pkgutil.FromContextOrDefault`?

Reply:
Sure I'll change these too. Within the controllers it is safe to assume we'll have the logger there but nbd to change these. Might add a linter rule to prevent using FromContextOrDiscard depending on what over usages remain